modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -15423,6 +15431,7 @@ rest-application=REST Application
 rest-client-logger-level=REST Client Logger Level
 rest-client-logger-level-help=Set the logging level for the Elasticsearch REST client.
 rest-endpoint=REST Endpoint
+rest-endpoints-are-created-for-the-new-parent-with-the-endpoints-of-remaining-children-grouped-under-it=REST endpoints are created for the new parent, with the endpoints of remaining children grouped under it
```

Comment:
REST endpoints are created for the new parent and the endpoints of remaining children will be grouped under it